pkg/cmd/server/api/types.go

```diff
@@ -325,6 +325,9 @@ type MasterNetworkConfig struct {
 	ClusterNetworkCIDR string
 	HostSubnetLength   uint
 	ServiceNetworkCIDR string
+	// ExternalIPNetworkCIDR controls what values are acceptable for the service external IP field. If empty, no externalIP
```

Comment:
> ExternalIP is a security risk, and we cannot ship with it enabled by
> default, therefore we must disable by default, and existing users will be
> broken.  We cannot allow through inaction for users to have a vulnerability
> here.  Release notes will be updated
> openshift/openshift-docs#1700

@liggitt @sdodson We'll probably get questions about this.

@sosiouxme Can you add a diagnostic check that looks at this setting, the openshift level, and the current services to find ones that violate this?  We can't do it client-side because we don't know the CIDR client-side.

@smarterclayton Be careful with this implementation.  As written, I think you'll make any controller that updates status end up having to deal with rejections.  Also, this makes wiring external services hard for end users because they don't know ahead of time what the CIDR is.  It also means we can't show them problems in `oc status`.